tests/testsuite/custom_target.rs

```diff
@@ -140,7 +145,10 @@ fn custom_bin_target() {
         .file("custom-bin-target.json", SIMPLE_SPEC)
         .build();
 
-    p.cargo("build --target custom-bin-target.json -v").run();
+    p.cargo("build")
```

Comment:
One drawback of it is that the argument list is spliited into multiple lines of indirection, making it harder to read at first glance.